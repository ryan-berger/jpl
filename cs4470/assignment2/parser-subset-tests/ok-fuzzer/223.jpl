read image "!-" to a
write image I(f2B_9P.(0, b1z., 42, I5R., C(O1j_5M.), B, N4L_(.231246679334, y, s, 67, N0H_(.69, J1V_, P4I_8s.6q_(w9p.(43, r(7))), 9, 14), t8e.6g.6I_(i4J., 9756, q5N_2U.(), c(), C(m7g.))), 70208805.8, 70306, s7P_(s(d2h_()), 8937, 1, H(), K4M_5r.(e, Z(E()), u(67., 2, p2C.(.658), N5z_(1, .4), s2I_5I_5y_8y.5h_(4.338), 659., 7, u4Q., R0b_, .81, 01, 2, x7U., 91), Z4g., G4C., 3601996., h, .3782, .935, 3., d(82783., T0Q_), 89463., 8., W, 69, x(), 4, y, 2, 66., 6858530, E2m_0G_(4, .2535), 73, G2Y_(), h(), F0P.(4, 35, s(O()), b5V_, U2D.(808865), 79, 8.), f(.5, o(e3T.())), .643503, z(), 6, 3, W2V., .4, 4, S(.86, 1, H6q.9B_(), 56169.6, y(), 78., 6, k1K., 13, 7, b0i_8M., O(.1), 4, 45932393., c(3), e5j_(X(K))), 0.7, 83, P, 916913.2, .25, 6, x2P_6i_, 8, .268, L, w4K_7V.8Y.0h.7J.2v_2K., L(), 82, n, 67, 234, 18, u, y2y.2d., f4V_5x.(6, i(A()), o(.4440231, u7z.3W.(W0a_)), G(9022., 3, .219)), .3, Q1k_(), 8, .869, 4, Q(9, q), q(.0), t, 6.4, y0J_8m_2o_7B.(), .0411, 83010667.6, 5454900.052, R9j_8w., N5t.0m.4C.(w()), S8t_1C., d(), 0.00, B1A.6y.0I_4F.(), 6099, I1f_(i5d.7M.9H_0B_(.5)), 1., I(.213), V(q4n.(), k(v5Y.()), 64.6738387371), 6, m7t.6L_(), 296537.495837, p, i5G_, c1O_9r_(), R(), .0, 8, 5, 297355, 1., 0, 5788.571, g, .8, 757756, 144., U(), 2, 67, .1074065, .64, 321)), F, 0, .69, z9B., .2673805, W, 22., z2G_(), .9228, .785, 4, 6, T, d(), n, f2C., 91, .86111477, 78983, X0N., 825., 9.875837761856914073, p4T_8G_0a_(), c1s.(90, U2x.8x_(O(H6L.())), 560., .0, 2560., .57, o(8)), 628033.)) to "^0"
assert 1445.359, "oD"
print ""
let T = 81385.
read image "Dc" to v
read image "l" to L0K_9U_
