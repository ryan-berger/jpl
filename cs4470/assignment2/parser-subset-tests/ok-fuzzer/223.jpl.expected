(ReadImageCmd "!-" (VarArgument a))
(WriteImageCmd
 (CallExpr
  I
  (CallExpr
   f2B_9P.
   (IntExpr 0)
   (VarExpr b1z.)
   (IntExpr 42)
   (VarExpr I5R.)
   (CallExpr C (VarExpr O1j_5M.))
   (VarExpr B)
   (CallExpr
    N4L_
    (FloatExpr 0)
    (VarExpr y)
    (VarExpr s)
    (IntExpr 67)
    (CallExpr
     N0H_
     (FloatExpr 0)
     (VarExpr J1V_)
     (CallExpr
      P4I_8s.6q_
      (CallExpr w9p. (IntExpr 43) (CallExpr r (IntExpr 7))))
     (IntExpr 9)
     (IntExpr 14))
    (CallExpr
     t8e.6g.6I_
     (VarExpr i4J.)
     (IntExpr 9756)
     (CallExpr q5N_2U.)
     (CallExpr c)
     (CallExpr C (VarExpr m7g.))))
   (FloatExpr 70208805)
   (IntExpr 70306)
   (CallExpr
    s7P_
    (CallExpr s (CallExpr d2h_))
    (IntExpr 8937)
    (IntExpr 1)
    (CallExpr H)
    (CallExpr
     K4M_5r.
     (VarExpr e)
     (CallExpr Z (CallExpr E))
     (CallExpr
      u
      (FloatExpr 67)
      (IntExpr 2)
      (CallExpr p2C. (FloatExpr 0))
      (CallExpr N5z_ (IntExpr 1) (FloatExpr 0))
      (CallExpr s2I_5I_5y_8y.5h_ (FloatExpr 4))
      (FloatExpr 659)
      (IntExpr 7)
      (VarExpr u4Q.)
      (VarExpr R0b_)
      (FloatExpr 0)
      (IntExpr 1)
      (IntExpr 2)
      (VarExpr x7U.)
      (IntExpr 91))
     (VarExpr Z4g.)
     (VarExpr G4C.)
     (FloatExpr 3601996)
     (VarExpr h)
     (FloatExpr 0)
     (FloatExpr 0)
     (FloatExpr 3)
     (CallExpr d (FloatExpr 82783) (VarExpr T0Q_))
     (FloatExpr 89463)
     (FloatExpr 8)
     (VarExpr W)
     (IntExpr 69)
     (CallExpr x)
     (IntExpr 4)
     (VarExpr y)
     (IntExpr 2)
     (FloatExpr 66)
     (IntExpr 6858530)
     (CallExpr E2m_0G_ (IntExpr 4) (FloatExpr 0))
     (IntExpr 73)
     (CallExpr G2Y_)
     (CallExpr h)
     (CallExpr
      F0P.
      (IntExpr 4)
      (IntExpr 35)
      (CallExpr s (CallExpr O))
      (VarExpr b5V_)
      (CallExpr U2D. (IntExpr 808865))
      (IntExpr 79)
      (FloatExpr 8))
     (CallExpr f (FloatExpr 0) (CallExpr o (CallExpr e3T.)))
     (FloatExpr 0)
     (CallExpr z)
     (IntExpr 6)
     (IntExpr 3)
     (VarExpr W2V.)
     (FloatExpr 0)
     (IntExpr 4)
     (CallExpr
      S
      (FloatExpr 0)
      (IntExpr 1)
      (CallExpr H6q.9B_)
      (FloatExpr 56169)
      (CallExpr y)
      (FloatExpr 78)
      (IntExpr 6)
      (VarExpr k1K.)
      (IntExpr 13)
      (IntExpr 7)
      (VarExpr b0i_8M.)
      (CallExpr O (FloatExpr 0))
      (IntExpr 4)
      (FloatExpr 45932393)
      (CallExpr c (IntExpr 3))
      (CallExpr e5j_ (CallExpr X (VarExpr K))))
     (FloatExpr 0)
     (IntExpr 83)
     (VarExpr P)
     (FloatExpr 916913)
     (FloatExpr 0)
     (IntExpr 6)
     (VarExpr x2P_6i_)
     (IntExpr 8)
     (FloatExpr 0)
     (VarExpr L)
     (VarExpr w4K_7V.8Y.0h.7J.2v_2K.)
     (CallExpr L)
     (IntExpr 82)
     (VarExpr n)
     (IntExpr 67)
     (IntExpr 234)
     (IntExpr 18)
     (VarExpr u)
     (VarExpr y2y.2d.)
     (CallExpr
      f4V_5x.
      (IntExpr 6)
      (CallExpr i (CallExpr A))
      (CallExpr o (FloatExpr 0) (CallExpr u7z.3W. (VarExpr W0a_)))
      (CallExpr G (FloatExpr 9022) (IntExpr 3) (FloatExpr 0)))
     (FloatExpr 0)
     (CallExpr Q1k_)
     (IntExpr 8)
     (FloatExpr 0)
     (IntExpr 4)
     (CallExpr Q (IntExpr 9) (VarExpr q))
     (CallExpr q (FloatExpr 0))
     (VarExpr t)
     (FloatExpr 6)
     (CallExpr y0J_8m_2o_7B.)
     (FloatExpr 0)
     (FloatExpr 83010667)
     (FloatExpr 5454900)
     (VarExpr R9j_8w.)
     (CallExpr N5t.0m.4C. (CallExpr w))
     (VarExpr S8t_1C.)
     (CallExpr d)
     (FloatExpr 0)
     (CallExpr B1A.6y.0I_4F.)
     (IntExpr 6099)
     (CallExpr I1f_ (CallExpr i5d.7M.9H_0B_ (FloatExpr 0)))
     (FloatExpr 1)
     (CallExpr I (FloatExpr 0))
     (CallExpr V (CallExpr q4n.) (CallExpr k (CallExpr v5Y.)) (FloatExpr 64))
     (IntExpr 6)
     (CallExpr m7t.6L_)
     (FloatExpr 296537)
     (VarExpr p)
     (VarExpr i5G_)
     (CallExpr c1O_9r_)
     (CallExpr R)
     (FloatExpr 0)
     (IntExpr 8)
     (IntExpr 5)
     (IntExpr 297355)
     (FloatExpr 1)
     (IntExpr 0)
     (FloatExpr 5788)
     (VarExpr g)
     (FloatExpr 0)
     (IntExpr 757756)
     (FloatExpr 144)
     (CallExpr U)
     (IntExpr 2)
     (IntExpr 67)
     (FloatExpr 0)
     (FloatExpr 0)
     (IntExpr 321)))
   (VarExpr F)
   (IntExpr 0)
   (FloatExpr 0)
   (VarExpr z9B.)
   (FloatExpr 0)
   (VarExpr W)
   (FloatExpr 22)
   (CallExpr z2G_)
   (FloatExpr 0)
   (FloatExpr 0)
   (IntExpr 4)
   (IntExpr 6)
   (VarExpr T)
   (CallExpr d)
   (VarExpr n)
   (VarExpr f2C.)
   (IntExpr 91)
   (FloatExpr 0)
   (IntExpr 78983)
   (VarExpr X0N.)
   (FloatExpr 825)
   (FloatExpr 9)
   (CallExpr p4T_8G_0a_)
   (CallExpr
    c1s.
    (IntExpr 90)
    (CallExpr U2x.8x_ (CallExpr O (CallExpr H6L.)))
    (FloatExpr 560)
    (FloatExpr 0)
    (FloatExpr 2560)
    (FloatExpr 0)
    (CallExpr o (IntExpr 8)))
   (FloatExpr 628033)))
 "^0")
(StmtCmd (AssertStmt (FloatExpr 1445) "oD"))
(PrintCmd "")
(StmtCmd (LetStmt (ArgLValue (VarArgument T)) (FloatExpr 81385)))
(ReadImageCmd "Dc" (VarArgument v))
(ReadImageCmd "l" (VarArgument L0K_9U_))
